examples/python/blackboard/README.md

```diff
@@ -0,0 +1,70 @@
+# Blackboard
+
+> [!CAUTION]
+> Every payload you transmit with iceoryx2 must be compatible with shared
+> memory. Specifically, it must:
+>
+> * be self contained, no heap, no pointers to external sources
+> * have a uniform memory representation, ensuring that shared structs have the
+>     same data layout
+>     * therefore, only `ctypes` and `ctypes.Structure` can be transferred
+> * not use pointers to manage their internal structure
+>
+> Any other python data type, except `ctypes` or `ctypes.Structure`s, like will
```

Comment:
This sentence is broken. Could you rephrase it to: ... ctypes.Structures, will likely cause undefined ...

And of course this sentence is in every python readme. Could you please adjust it.